.vscode/settings.json

```diff
@@ -2,9 +2,9 @@
 {
   "debug.internalConsoleOptions": "openOnSessionStart",
   "editor.codeActionsOnSave": {
-    "editor.action.fixAll": true,
-    "source.fixAll.eslint": true,
-    "source.fixAll.markdownlint": true
+    "editor.action.fixAll": "explicit",
```

Comment:
do you intend to change this setting for everyone?

Reply:
Interesting... I don't recall making that change.  I'll revert.

Reply:
Looks like this can't be helped with the latest version of VS Code - https://stackoverflow.com/questions/77637621/vscode-workspace-settings-change-on-its-own